### DISCUSSION TITLE
Uncaught (in promise) TypeError: n is not a function

### DISCUSSION THREAD

**Discussion by chef-plankton** [WalletConnect/walletconnect-monorepo#4071] [Developer Support]
Hello
I'm getting error in my domain but not in other domains.
my main domain is : https://payment.paybybit.net/en/
where is get error.
I'm not getting error in other domains like : https://paybybit.vercel.app/en/
maybe my domain get banned?
![Screenshot 2023-12-20 234718](https://github.com/WalletConnect/walletconnect-monorepo/assets/107023631/daeb3432-ce4e-401a-8186-1b5601c2bcce)

**Comment by glitch-txs** [comment]
I think that's from an extension wallet, unrelated to WalletConnect
